manifest_staging/charts/workload-identity-webhook/templates/azure-wi-webhook-controller-manager-deployment.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   namespace: '{{ .Release.Namespace }}'
 spec:
   replicas: {{ .Values.replicaCount }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
```

Comment:
Gate this with an `if` check similar to

https://github.com/Azure/azure-workload-identity/blob/b0c4e018378fbea472aa2f82501a96ab681220d1/manifest_staging/charts/workload-identity-webhook/templates/azure-wi-webhook-controller-manager-poddisruptionbudget.yaml#L12-L14